engine/collection/ingest/engine.go

```diff
@@ -54,13 +54,12 @@ func New(
 		access.NewProtocolStateBlocks(state),
 		chain,
 		access.TransactionValidationOptions{
-			Expiry:                       flow.DefaultTransactionExpiry,
-			ExpiryBuffer:                 config.ExpiryBuffer,
-			AllowUnknownReferenceBlockID: config.AllowUnknownReference,
```

Comment:
Do we also want to remove this field in the definition of `access.TransactionValidationOptions`?

Reply:
I do still think it's useful to keep the option there. I can see uses, like allowing collection nodes to temporarily accept and validate transactions with an unknown reference block.